addons/website/static/src/scss/website.scss

```diff
@@ -2446,3 +2446,43 @@ input[value*="data-oe-translation-initial-sha"] {
 .o_dropdown_without_offset {
     transform: none !important;
 }
+
+// Subtle input-group-text
+// This can be set to an input-group-text placed directly before or after the input.
+.input-group-text.input-group-text-subtle {
+    @include transition($input-transition);
+    border-color: $input-border-color;
+    background: $input-bg;
+    color: $input-color;
+
+    > * {
+        color: rgba($input-color, 0.6);
+    }
+
+    .is-invalid ~  &,
+    &:has(~ .is-invalid) {
+        border-color: $form-feedback-invalid-color;
+    }
+
+    .is-invalid ~  & + .input-group-text {
```

Comment:
Nitpicking but my convention for CSS in general is:
```scss
.a {
    rules;

    .d {
    
        .c {
            rules;
        }
        .d {
            rules;
        }
    }
    .e {
        rules;
    }
}
```
=> Blank line after rules or parent selector with not rules, not between nested rules.

Feel free not to follow it, maybe it's just me :wink: